tilequeue/command.py

```diff
@@ -150,7 +157,12 @@ def make_seed_tile_generator(cfg):
     else:
         top_tiles = ()
 
-    combined_tiles = chain(all_tiles, metro_extract_tiles, top_tiles)
+    custom_tiles = tile_generator_for_multiple_bounds(
```

Comment:
Can this be guarded with an explicit check like the others? Set it to an empty list/tuple if the bounds and/or zooms aren't configured.